patches/minecraft/net/minecraft/client/renderer/entity/RenderItem.java.patch

```diff
@@ -9,7 +9,61 @@
          this.func_175041_b();
      }
  
-@@ -226,7 +226,7 @@
+@@ -153,7 +153,7 @@
+ 
+                 if (p_180454_1_.func_77962_s())
+                 {
+-                    this.func_180451_a(p_180454_2_);
++                    this.func_180451_a(p_180454_2_, p_180454_1_);
+                 }
+             }
+ 
+@@ -161,33 +161,18 @@
+         }
+     }
+ 
+-    private void func_180451_a(IBakedModel p_180451_1_)
++    private void func_180451_a(IBakedModel p_180451_1_, ItemStack stack)
+     {
+-        GlStateManager.func_179132_a(false);
```

Comment:
Fair enough. I'm no where near the level of being able to write something that'll account for all possible calls to render items in the pipeline, cache them, and efficiently batch the lot of them together for the single draw call, so I'll definitely be leaving things like this to the more knowledgeable.